lib/config/options/index.ts

```diff
@@ -2416,17 +2416,18 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'concurrentRequestLimit',
-    description: 'Limit concurrent requests per host.',
+    description:
+      'Limit concurrent requests per host. Set to `-1` for no limit. Default value is 5 concurrent requests.',
     type: 'integer',
     stage: 'repository',
     parents: ['hostRules'],
-    default: null,
     cli: false,
     env: false,
   },
   {
     name: 'maxRequestsPerSecond',
-    description: 'Limit requests rate per host.',
+    description:
+      'Limit requests rate per host. Set to `-1` for no limit. Default value is 5 requests per second.',
```

Comment:
This one has default=0, seems like it should be removed, or changed to 5 if you agree with my suggestion above?

Reply:
Yes, agree, added it

Reply:
So what does a value of 0 mean if the user sets it?

Reply:
It's same as -1, i.e. no limit

Reply:
-1 just seemed more consistent for our configuration practices

Reply:
Not sure why, though. Because we usually use 0 for defaults